l3kernel/l3prg.dtx

```diff
@@ -76,7 +76,24 @@
 % \section{Defining a set of conditional functions}
 % \label{sec:l3prg:new-conditional-functions}
 %
-% \begin{function}[updated = 2022-11-01]
+% \begin{function}[added = 2024-11-22]
+%   {
+%     \prg_new_conditional:Npn, \prg_set_conditional:Npn, \prg_gset_conditional:Npn,
+%     \prg_new_conditional:Nn,  \prg_set_conditional:Nn, \prg_gset_conditional:Nn
+%    }
+%   \begin{syntax}
+%     \cs{prg_new_conditional:Npn} \cs[no-index]{\meta{name}:\meta{arg spec}} \meta{parameters} \Arg{code} \\
+%     \cs{prg_new_conditional:Nn} \cs[no-index]{\meta{name}:\meta{arg spec}} \Arg{code}
+%   \end{syntax}
+%   These functions create a family of conditionals using the same
+%   \Arg{code} to perform the test created. Those conditionals are
+%   expandable if \meta{code} is. The \texttt{new} versions check
```

Comment:
As now `_p` predicates are automatically created, maybe the `<code>` must be expandable here?

The same argument holds for the docs for `\prg_new_conditional:Npnn` and friends below.

Reply:
I now see that in some cases engine-expandable (no `\protected` prefix) TF-conditionals whose codes are not fully expandable may be useful (to `f`-type expansion lovers?).